### ISSUE TITLE
Fresh install of speki yields error 106 at compile time

### ISSUE THREAD

**Issue by DiMeneses** [TBS1996/speki#10]
I tried to install on Arch with: `yay -S speki`

Result:
```sh
   Compiling speki v0.1.8 (/home/vasco/.cache/yay/speki/src/speki-0.1.8)
error[E0106]: missing lifetime specifier
  --> src/main.rs:40:26
   |
40 |     const DEFAULTCONFIG: &str = r#"
   |                          ^ expected named lifetime parameter
   |
help: consider using the `'static` lifetime
   |
40 |     const DEFAULTCONFIG: &'static str = r#"
   |                           +++++++
help: consider introducing a named lifetime parameter
   |
39 ~ impl<'a> SpekiPaths {
40 ~     const DEFAULTCONFIG: &'a str = r#"
   |

For more information about this error, try `rustc --explain E0106`.
error: could not compile `speki` due to previous error
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: speki
```

**Comment by DiMeneses** [comment]
Unrelated to this issue but the crates.io demo link is not displaying like it did yesterday:

https://crates.io/crates/speki/0.1.8

vs

https://crates.io/crates/speki/0.1.7

**Comment by TBS1996** [comment]
This is odd, I am also using arch and yay, I tried uninstalling, deleting config file and local files and installing and it works. 

anyway  I did as the compiler on your end told me, the version is now 0.1.9, try updating it and see if it works!

about the README, I found out github can embed videos directly but didnt think about how it doesnt show up on crates.io... ill fix later X) 

Hope it works for you now!
